llvm/include/llvm/ADT/STLExtras.h

```diff
@@ -1994,6 +1994,10 @@ auto unique(Range &&R, Predicate P) {
   return std::unique(adl_begin(R), adl_end(R), P);
 }
 
+template <typename Range> auto unique(Range &&R) {
+  return std::unique(adl_begin(R), adl_end(R));
+}
```

Comment:
I understand this is just moved code, but all functions in STLExtras have to be documented, even if they just defer to the stl documentation; see an example on the function just below this one

Reply:
Done, in the new PR (requested by dwblaikie): https://github.com/llvm/llvm-project/pull/82312